lib/suse/connect/client.rb

```diff
@@ -240,17 +252,20 @@ def registered?
       # Refreshing the service instead to remove the repos of deregistered product.
       def remove_or_refresh_service(service)
         if service.name == 'SMT_DUMMY_NOREMOVE_SERVICE'
+          log.info '-> Refreshing service ...'
           Zypper.refresh_all_services
         else
+          log.info '-> Removing service from system ...'
           System.remove_service service
         end
       end
 
       def print_success_message(product, action: 'Registered')
-        log.info "#{action} #{product.identifier} #{product.version} #{product.arch}"
+        log.info "\n#{action} #{product.identifier} #{product.version} #{product.arch}".bold
         log.info "Rooted at: #{@config.filesystem_root}" if @config.filesystem_root
         log.info "To server: #{@config.url}" if @config.url
```

Comment:
"Deregistered *to* server" reads wrong, I'd use *from* in this case.  

I also think passing the verb as a string parameter to the message is going to be problematic when we have to translate SUSEConnect output, so perhaps just pass the action as a symbol and construct the output strings entirely within the function?

Reply:
I understand the issue with the translations, but how does passing a symbol clean this up specifically?